### ISSUE TITLE
What best param to tune? Or increasing number of epochs?

### ISSUE THREAD

**Issue by jpoullet2000** [llSourcell/keras_explained#2]
After 60 epochs using the exact same code as yours I get the results below which is not very convincing to say the least ;)... ok the loss function has decreased... ;)

Wondering whether you'd rather recommend to increase the number of epochs or to change some metaparams?

Thanks for the insight.

```
corpus length: 600893
total chars: 57
nb sequences: 200285
Vectorization...
Build model...
Epoch 1/60
200285/200285 [==============================] - 54s 270us/step - loss: 1.9812

...

Epoch 60/60
200285/200285 [==============================] - 58s 289us/step - loss: 1.3503
----- Generating text after Epoch: 59
----- diversity: 0.2
----- Generating with seed: "rt of a god beneath whose invisible infl"
rt of a god beneath whose invisible inflictification of the consciousness of the surve of the south and the portally the prosent and sense of the standarded to the sense of the stronger the consciousness of the stronger of the surves of the surves of the last the consciousness of the stronger the stronger the stronger the surves of a conscience of the sense and the stronger the powerful and conscience of the stronger the sense of the st
----- diversity: 0.5
----- Generating with seed: "rt of a god beneath whose invisible infl"
rt of a god beneath whose invisible infliciated in the english for the sense of concealed to the experience and solitary of every deceived in the new order to it is a desire to be proavian and attempt to contemptation in the sense of its old the merely and old and the newsly of the consciousng and more will an anti-all the extendity of the soul is the approvogius of the consciousness of the properage itself of the fact of the standarded
----- diversity: 1.0
----- Generating with seed: "rt of a god beneath whose invisible infl"
rt of a god beneath whose invisible inflicious andigbariated and fausishessy range suppreciate , the victomy into theology, the influence for the contemptuar?--we are sacriletion of the use in it sublimation
largely will appears where to gade with
represente".




hattering every ment, hi
so vait grkaks of the
powerfful sensed
of hagh.

219. "itrageologic, hour consceourous classific and
noteousness of comparely. in the people done. it 
----- diversity: 1.2
----- Generating with seed: "rt of a god beneath whose invisible infl"
rt of a god beneath whose invisible inflictible: ho a right siets,
kneine of thinkis
husitinal,
the songer of
sake living; as the learn metangeset just of all them sirf, to ale, motivele of your metaphysics to had were to self euddness at has been possible. to well be protecturant on former" oneyring to be religiots. i revenge-in"migne--the powerful lode of leaders, curioar case hards its believe and rational thoses of the fundsedobls o
```
